invenio_communities/searchapp.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-2021 CERN.
+#
+# Invenio-App-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Configuration helper for React-SearchKit."""
+
+from functools import partial
+
+from flask import current_app
+
+from invenio_search_ui.searchconfig import search_app_config
+
+
+def search_app_context():
+    """Search app context processor."""
+    return {
+        'search_community_requests_config': partial(
+            search_app_config,
+            'COMMUNITIES_SEARCH_REQUESTS',
+            current_app.config['REQUESTS_FACETS'],
+            current_app.config['COMMUNITIES_SORT_OPTIONS'],
+            '/api/requests',
```

Comment:
hardcoded URL, please check with Javier if it can be avoided (he has the same config)

Reply:
It can be injected from the `routes` config, as you have added it above.